ttnn/cpp/ttnn/operations/reduction/generic/generic_reductions.hpp

```diff
@@ -29,128 +26,11 @@ template <ReduceType reduce_type>
 struct Reduce {
     static Tensor operator()(
         const Tensor& input_tensor_arg,
-        const std::optional<std::variant<int, std::vector<int>>>& dim_arg,
-        const bool keepdim,
+        const std::optional<std::variant<int, std::vector<int>>>& dim_arg = std::nullopt,
+        const bool = true,
```

Comment:
please review this

Reply:
thx, I'm not sure how I ended up with this

Reply:
happens